tests/test_patterns.py

```diff
@@ -234,6 +234,23 @@ def test_bd_phone_number_valid(self):
         for phone_no in valid_numbers:
             self.assertTrue(re.match(num_pattern, phone_no), 'Bangladeshi phone number: ' + phone_no)
 
+    def test_ru_phone_number_valid(self):
+        num_pattern = self.patterns.PHONE_NUMBER['RU']
+
+        valid_numbers = ['+74951234567', '84951234567', '74951234567', '+7 495 123 45 67', '8(926)123-45-67',
```

Comment:
Can you, please use decorator  `@pytest.mark.parametrize`?

How to use it:

```python
@pytest.mark.parametrize(
    'valid_numbers', [
        '+74951234567', 
        '84951234567', 
        '74951234567',
        # ...
    ]
)
def test_ru_phone_number_valid(self, valid_numbers):
```

Reply:
Are u sure that we should import new module ?

Reply:
@kl09 Oh, god. Sorry. I have confused this project with another one.